tests/integration/ethernet_mtu_test.py

```diff
@@ -135,6 +135,12 @@ def test_decrease_to_lower_than_min_ipv6_iface_mtu(eth1_with_ipv6):
     assertlib.assert_state(original_state)
 
 
+def test_decrease_mtu_to_lower_than_1280(eth1_up):
```

Comment:
Since IPv6 being disabled is important here it should be mentioned in the test name, maybe `test_mtu_without_ipv6` and then parametrize it with 1279, 926 and 576 to test the edge cases and a middle value.

Even better would be to introduce a constant for 1280 such as `MINIMAL_IPV6_MTU` and then use `MINIMAL_IPV6_MTU - 1`.

Reply:
Done! Thanks!